packages/eas-cli/src/run/android/aapt.ts

```diff
@@ -0,0 +1,59 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import glob from 'fast-glob';
+import path from 'path';
+import unixify from 'unixify';
+
+import Log from '../../log';
+import { getAndroidSdkRootAsync } from './sdk';
+
+async function aaptAsync(...options: string[]): Promise<SpawnResult> {
+  try {
+    return await spawnAsync(await getAaptExecutableAsync(), options);
+  } catch (error: any) {
+    if (error.stderr) {
+      Log.error(error.stderr);
+    }
+    throw error;
+  }
+}
+
+export async function getAaptExecutableAsync(): Promise<string> {
+  const sdkRoot = await getAndroidSdkRootAsync();
+  if (!sdkRoot) {
+    Log.debug('Failed to resolve the Android SDK path, falling back to global aapt executable');
+    return 'aapt';
+  }
+  const aaptPaths = await glob(
+    unixify(
+      path.join(sdkRoot, 'build-tools', '**', process.platform === 'win32' ? 'aapt.exe' : 'aapt')
+    )
+  );
```

Comment:
```suggestion
  const aaptPaths = await glob(
      path.posix.join('build-tools/**', process.platform === 'win32' ? 'aapt.exe' : 'aapt')`,
      { cwd: sdkRoot, absolute: true }
  );
```
unixify is loosing information e.g. both `C:\\sth` and `D:\\sth` would be resolved to `/sth`. I have no idea how fast-glob is handling patterns that are absolute path on windows, but even if it works correctly we should not rely on that.

Reply:
Agreed with Wojtek, we shouldn't need to use `unixify`.